src/components/Common/EditField.vue

```diff
@@ -44,10 +44,12 @@
 
 <script lang="ts">
   import { Component, Vue, Mixins, Prop } from 'vue-property-decorator';
+  // TODO https://github.com/eternagame/eternagame.org/issues/17 improve typing
```

Comment:
I don't know that these comments are more helpful than distracting (presumably the person cleaning up #17 can just do a find-all for `@ts-ignore`

Reply:
noted